src/main/webapp/WEB-INF/pages/orderOverview.jsp

```diff
@@ -0,0 +1,70 @@
+<%@ page contentType="text/html;charset=UTF-8" %>
```

Comment:
checkout.jsp and orderOverview.jsp files have duplicate code fragments. For the future: think of how to avoid this.